### PR TITLE
Signup UI

### PR DESCRIPTION
**Pull request by Jhoscar22**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [x] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Signin UI

* **What is the current behavior?** (You can also link to an open issue here)
No signin page

* **What is the new behavior (if this is a feature change)?**
Now there is a signin UI at the /signin route

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No

* **Other information**:

### CODE REVIEW

File: data-dashboard/pages/signup.tsx [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+
+const LinkStyle = {
+  marginLeft: '44px',
+  fontFamily: 'Verdana, Geneva, Tahoma, sans-serif',
+  color: '#0F62FE',
+  fontSize: '14px',
+  fontWeight: 'normal',
+};
+
+function SignIn() {
```
Comment (Int-Angel) [inline data-dashboard/pages/signup.tsx:83]: I think this should be SignUp

Comment (DavBot02) [inline data-dashboard/pages/signup.tsx:83]: Signup

**Review by Int-Angel — CHANGES_REQUESTED** [review]
I think you should change all "signin" for "signup"

File: data-dashboard/pages/signup.tsx (hunk 1 above)
Comment (Jhoscar22) [reply data-dashboard/pages/signup.tsx]: Done!

**Review by Int-Angel — APPROVED** [review]
LGTM!

**Review by DavBot02 — APPROVED** [review]

**Pull request merged**